preprocess-and-train.py

```diff
@@ -2,15 +2,21 @@
 import cv2
 from model import generate_model
 
-TRAINING_DATA_DIR = "training_data"
+TRAINING_DATA_DIR = "C:\\Users\\Manish Sehrawat\\Downloads\\training_data"
 
 # here write code to pre process the images as required
 # but make sure to have a copy of the original training data
 # or that you write your code in such a way that you pre-process on the fly
 # pre-processing on the fly is possible because this is small project and
 # not a lot of images will be there as well.
 def preprocess(img):
-    # your code goes here
+    
+    #Image Resizing
+    width = 225
+    height = 225
+    dimensions = (width,height)
+    img = cv2.resize(img,dimensions,interpolation = cv2.INTER_LINEAR)
+    img = cv2.GaussianBlur(img,(5,5),0)
```

Comment:
what other preprocessing techniques did you try?
and what were the results of those techniques? @ManishSehrawat